shared/src/main/scala/com/raquo/domtypes/codegen/CanonicalDefGroups.scala

```diff
@@ -25,6 +25,10 @@ class CanonicalDefGroups {
     "SVG Tags" -> SvgTagDefs.defs,
   )
 
+  val mathMlTagsDefGroups: List[(String, List[TagDef])] = List(
+    "MathMl Tags" -> MathMlTagDefs.defs,
```

Comment:
"MathMl" should be capitalized as "MathML" in the key here, to match other keys that capitalize e.g. SVG.